dockers/docker-snmp-sv2/Dockerfile.j2

```diff
@@ -1,48 +1,40 @@
 FROM docker-base
 
-COPY \
+COPY [ \
 {% for deb in docker_snmp_sv2_debs.split(' ') -%}
-debs/{{ deb }}{{' '}}
-{%- endfor -%}
-debs/
+    "debs/{{ deb }}",
+{%- endfor %} \
+    "/debs/"]
 
-COPY python-wheels /python-wheels
+COPY python-wheels/sswsdk-*-py3-*.whl /python-wheels/
+COPY python-wheels/asyncsnmp-*-py3-*.whl /python-wheels/
 
 # enable -O for all Python calls
 ENV PYTHONOPTIMIZE 1
 
-## Pre-install the fundamental packages
-## Install Python SSWSDK (SNMP subagent dependency)
-## Install SNMP subagent
-## Clean up
-RUN apt-get update && apt-get install -y libmysqlclient-dev libmysqld-dev libperl-dev libpci-dev libpci3 libsensors4 libsensors4-dev libwrap0-dev
-
-# Dependencies for sonic-cfggen                                                                                                                                                           
-RUN apt-get install -y python-lxml python-jinja2 python-netaddr python-ipaddr python-yaml 
-
-RUN dpkg -i \
-{% for deb in docker_snmp_sv2_debs.split(' ') -%}
-debs/{{ deb }}{{' '}}
-{%- endfor %}
-
-RUN rm -rf /debs
-
+# install supervisor
+# install libsnmp30 dependencies
+# install dependencies for sonic-cfggen
+# install libpython3.6-dev and pip dependencies
+# TODO: remove libpython3.6-dev, its and pip's dependencies if we can get pip3 directly
```

Comment:
libpython3.6-dev's (dependencies)

---
In reply to: [99458291](https://github.com/Azure/sonic-buildimage/pull/245#discussion_r99458291) [](ancestors = 99458291)